spec/system/school_alerts_spec.rb

```diff
@@ -34,17 +34,31 @@
       expect(page.has_content?(poor_alert_summary)).to_not be true
     end
 
-    it 'can show a single alert' do
-      alert_summary = 'Summary of the alert'
-      Alert.create(alert_type: gas_fuel_alert_type, run_on: gas_date, school: school, status: :good, data: { detail: [], rating: 10.0}, summary: alert_summary)
+    describe 'Find Out More' do
```

Comment:
We might want to better structure our system tests when we have different roles doing different things.